src/AzurePipelinesToGitHubActionsConverter.Core/Conversion/VariablesProcessing.cs

```diff
@@ -314,5 +398,91 @@ private List<string> FindPipelineParametersInString(string text)
 
             return list;
         }
+
+        public MatchCollection FindPipelineVariables(string input)
```

Comment:
Can these be consolidated to not repeat the expression?  i.e. Can they overload and just pass a string.empty, then the string.format() will replace the {0} and it's otherwise exactly the same?

Reply:
Thanks, was on my mental list. 2nd version wasn't even being used currently, but I refactored so they chain together.